Sources/KlaviyoSwift/StateManagement.swift

```diff
@@ -217,7 +217,7 @@ struct KlaviyoReducer: ReducerProtocol {
                 state.pendingRequests.append(.pushToken(pushToken, enablement))
                 return .none
             }
-            guard state.shouldSendTokenUpdate(newToken: pushToken, enablement: enablement) else {
+            if !state.shouldSendTokenUpdate(newToken: pushToken, enablement: enablement) {
```

Comment:
This is a personal preference change. I find it easier to read if else for conditionals over guard but happy to revert since I don't have a good reason to change it.